shared-helpers/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "dependencies": {
     "@bloom-housing/ui-components": "12.4.0",
-    "@bloom-housing/ui-seeds": "1.17.0",
+    "@bloom-housing/ui-seeds": "^1.18.1",
```

Comment:
nit: Ideally we will only be updating dependencies in core so our package and yarn files stay in sync and we don't run into any weirdness with deployments because of discrepancies

Reply:
Reverted! 🫡